main/modes/games/cGrove/Garden/cg_Grove.c

```diff
@@ -747,9 +749,9 @@ void cg_runGrove(cGrove_t* cg, int64_t elapsedUS)
                 if (evt.down && evt.button & PB_DOWN)
                 {
                     cg->grove.tutorialPage++;
-                    if (cg->grove.tutorialPage > 6)
+                    if (cg->grove.tutorialPage > 5)
```

Comment:
This likely works fine, but I try not to use hardcoded numbers. An option here would be to make an enum like
```c
typedef enum {
    TP_WELCOME,
    TP_NAVIGATION,
    TP_A_B_BUTTONS,
    TP_DONUTS,
    TP_GETTING_STARTED,
    NUM_TUTORIAL_PAGES,
} tutorialPage_t;
```
and then use `NUM_TUTORIAL_PAGES` for counting

Reply:
This is called me getting lazy. Sorry.

Reply:
The Spar tutorial is largely copy-pasted, so it has the same issue

Reply:
I too am lazy, which is why I don't trust myself to keep indices updated across projects manually!